setup.sh

```diff
@@ -47,39 +47,24 @@ function check_dependencies()
   local pip_package_list=''
   local cmd=''
   local distro
-
+  debian=("apt list --installed | cut -d '/' -f1" "apt install -y")
```

Comment:
btw, this commit sequence is better:

- Commit 1: This code refactor.
- Commit 2: Add Gentoo part.

Reply:
I'll do it